src/test/java/uk/ac/ebi/eva/pipeline/jobs/VariantAggregatedConfigurationTest.java

```diff
@@ -18,188 +18,153 @@
-        String input = VariantAggregatedConfigurationTest.class.getResource(FILE_AGGREGATED).getFile();
-        String dbName = VALID_LOAD;
-
-        pipelineOptions.put("input.vcf", input);
-        variantOptions.put(VariantStorageManager.DB_NAME, dbName);
-
-        VariantSource source = (VariantSource) variantOptions.get(VariantStorageManager.VARIANT_SOURCE);
-
-        variantOptions.put(VariantStorageManager.VARIANT_SOURCE, new VariantSource(
-                input,
-                source.getFileId(),
-                source.getStudyId(),
-                source.getStudyName(),
-                source.getType(),
-                source.getAggregation()));
-
-        JobExecution execution = jobLauncher.run(job, JobTestUtils.getJobParameters());
-
-        assertEquals(input, pipelineOptions.getString("input.vcf"));
-        assertEquals(ExitStatus.COMPLETED, execution.getExitStatus());
 
         // check ((documents in DB) == (lines in transformed file))
         VariantStorageManager variantStorageManager = StorageManagerFactory.getVariantStorageManager();
         VariantDBAdaptor variantDBAdaptor = variantStorageManager.getDBAdaptor(dbName, null);
         VariantDBIterator iterator = variantDBAdaptor.iterator(new QueryOptions());
 
-        String outputFilename = JobTestUtils.getTransformedOutputPath(Paths.get(FILE_AGGREGATED).getFileName(),
-                variantOptions.getString("compressExtension"), pipelineOptions.getString("output.dir"));
-        long lines = JobTestUtils.getLines(new GZIPInputStream(new FileInputStream(outputFilename)));
-
         Assert.assertEquals(JobTestUtils.count(iterator), lines);
 
-        // check stats aren't loaded
-        assertTrue(variantDBAdaptor.iterator(new QueryOptions()).next().getSourceEntries().values().iterator().next().getCohortStats().isEmpty());
+        // check that stats are loaded properly
+        assertFalse(variantDBAdaptor.iterator(
+                new QueryOptions()).next().getSourceEntries().values().iterator().next().getCohortStats().isEmpty());
     }
 
     @Test
-    public void validLoadStats() throws JobParametersInvalidException, JobExecutionAlreadyRunningException,
-            JobRestartException, JobInstanceAlreadyCompleteException, IllegalAccessException, ClassNotFoundException,
-            InstantiationException, StorageManagerException, IOException {
-        String input = VariantAggregatedConfigurationTest.class.getResource(FILE_AGGREGATED).getFile();
-        String dbName = VALID_LOAD_STATS;
-
-        pipelineOptions.put("input.vcf", input);
-        variantOptions.put(VariantStorageManager.DB_NAME, dbName);
-
-        variantOptions.put("includeStats", true);
-
-        VariantSource source = (VariantSource) variantOptions.get(VariantStorageManager.VARIANT_SOURCE);
-
-        variantOptions.put(VariantStorageManager.VARIANT_SOURCE, new VariantSource(
+    public void aggregationNoneOptionShouldNotLoadStats() throws Exception {
```

Comment:
In this case is checking that the stats are not loaded (.getCohortStats().isEmpty()) when we use Aggregation.NONE over an aggregated VCF. If this is not important or doesn't make sense then I suppose I can delete the test